index.js

```diff
@@ -1,16 +1,51 @@
 require('dotenv').config();
 const path = require('path');
 const express = require('express');
+const cors = require('cors');
```

Comment:
You don't seem to be using cors at all in this file. You probably imported it in anticipation of use in the future, but I just wanted to make sure you're aware!

Reply:
Also a good catch. I was trying to use this earlier as a workaround before LeiLei helped me with precise syntax.